GVFS/GVFS.Platform.Windows/WindowsFileSystem.cs

```diff
@@ -117,6 +117,58 @@ public bool IsSocket(string fileName)
             return false;
         }
 
+        public void CreateDirectoryAccessibleByAuthUsers(string directoryPath)
+        {
+            if (Directory.Exists(directoryPath))

```

Comment:
Do we want to emit a warning if the directory has already been created but does _not_ have the correct ACLs?